### PR TITLE
Update build.gradle

### PR DESCRIPTION
**Pull request by yedidyak**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -16,9 +16,10 @@ def _minSdkVersion      = safeExtGet('minSdkVersion', 16)
 buildscript {
     repositories {
         google()
+        mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.4.1'
+        classpath "com.android.tools.build:gradle:$androidGradlePluginVersion"
```
Comment (Inbal-Tish) [inline android/build.gradle]: Let's remove this change for now

Comment (yedidyak) [reply android/build.gradle]:
```suggestion
        classpath "com.android.tools.build:gradle:3.4.1"
```

Comment (Inbal-Tish) [reply android/build.gradle]: It's your PR, you need to commit this

Comment (yedidyak) [reply android/build.gradle]: I did, it's commited

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**